pkg/storage/driver/sql.go

```diff
@@ -150,7 +150,7 @@ func (s *SQL) Get(key string) (*rspb.Release, error) {
 	// Get will return an error if the result is empty
 	err := s.db.Get(&record, "SELECT body FROM releases WHERE key = $1", key)
 	if err != nil {
-		s.Log("got SQL error when getting release %s: %v", key, err)
+		s.Log("got sql error when getting release %s: %v", key, err)
```

Comment:
SQL is a formal name, so this should be reverted.

Reply:
Got it. 
SQL should be reverted but othere formal names e.g. "Job", "Service" are inconsistent. In some log, it is lowercase.
This may be a trival "change" and if it is unnecessary, I will close the pr. :-)